Cargo.toml

```diff
@@ -18,7 +18,7 @@ resolver = "2"
 version = "0.10.0"
 authors = ["thin-edge.io team <info@thin-edge.io>"]
 edition = "2021"
-rust-version = "1.64"
+rust-version = "1.65"
```

Comment:
I had a convenient use case for let-else in the tests, but I can easily change this to match if this version bump is unwanted.

Reply:
I've created https://github.com/thin-edge/thin-edge.io/pull/1885 in an effort to fix the integration test workflow as that will need to be merged separately to get the integration tests to run here.